hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/Utils.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.fs.azurebfs.utils;
+
+public final class Utils {
+
+  private Utils() {
+  }
+
+  public static boolean isEmpty(String str) {
```

Comment:
normally we use commons-language isEmpty here; this is quietly different in that it also does trimming.

Can you 
* name it as such to avoid confusing people, e.g. "isEmptyAfterTrimming"
* add a test

Reply:
+ javadocs for this and the class, so checkstyle is happy

Reply:
Used apache commons-language utility method